src/components/app-content-switcher/AppContentSwitcher.jsx

```diff
@@ -0,0 +1,38 @@
+import { Typography } from '@mui/material'
+import Tooltip from '@mui/material/Tooltip'
+import Switch from '@mui/material/Switch'
+import Stack from '@mui/material/Stack'
+import { defaultStyles } from '~/components/app-content-switcher/AppContentSwitcher.styles'
+
+const AppContentSwitcher = (props) => {
+  const { active, handleChange, tooltipRight, tooltipLeft, spacing, leftText, rightText, colorActive, colorInActive, typographyVariant, styles } = props
```

Comment:
do outside two objects , right and left:{tooltip, text, and all what you need}

Reply:
refactored